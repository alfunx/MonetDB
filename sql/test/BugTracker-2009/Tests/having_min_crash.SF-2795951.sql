select schema_id from tables group by schema_id having min(schema_id);
