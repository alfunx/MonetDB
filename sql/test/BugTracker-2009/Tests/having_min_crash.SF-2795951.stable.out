stdout of test 'having_min_crash.SF-2795951` in directory 'sql/test/BugTracker-2009` itself:


# 21:57:32 >  
# 21:57:32 >   mserver5 "--config=/ufs/niels/scratch/rc/Linux-x86_64/etc/monetdb5.conf" --debug=10 --set gdk_nr_threads=0 --set "monet_mod_path=/ufs/niels/scratch/rc/Linux-x86_64/lib/MonetDB5:/ufs/niels/scratch/rc/Linux-x86_64/lib/MonetDB5/lib:/ufs/niels/scratch/rc/Linux-x86_64/lib/MonetDB5/bin" --set "gdk_dbfarm=/ufs/niels/scratch/rc/Linux-x86_64/var/MonetDB5/dbfarm"    --set mapi_open=true --set xrpc_open=true --set mapi_port=36965 --set xrpc_port=41502 --set monet_prompt= --trace  "--dbname=mTests_src_test_BugTracker-2009" --set mal_listing=0 "--dbinit= include sql;" ; echo ; echo Over..
# 21:57:32 >  

# MonetDB server v5.12.0, based on kernel v1.30.0
# Serving database 'mTests_src_test_BugTracker-2009', using 4 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 64bit OIDs dynamically linked
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008-2009 MonetDB B.V., all rights reserved
# Visit http://monetdb.cwi.nl/ for further information
# Listening for connection requests on mapi:monetdb://alf.ins.cwi.nl:36965/
# MonetDB/SQL module v2.30.0 loaded

Ready.
#function user.main():void;
#    clients.quit();
#end main;


# 21:57:32 >  
# 21:57:32 >  mclient -lsql -umonetdb -Pmonetdb --host=alf --port=36965 
# 21:57:32 >  

% .tables # table_name
% schema_id # name
% int # type
% 1 # length

# 21:57:32 >  
# 21:57:32 >  Done.
# 21:57:32 >  

